examples/example_with_all.cpp

```diff
@@ -1,4 +1,4 @@
-#include "../amalgamate/crow_all.h"
+#include "../build/crow_all.h"
```

Comment:
`../crow_all.h`?

Reply:
`../crow_all.h` would imply copying crow_all to the root directory and using it from there. I can do that if you want, but it's not that necessary.